rust/frontend/tvm-sys/LICENSE

```diff
@@ -0,0 +1,201 @@
+                              Apache License
```

Comment:
strictly speaking, this should symlink to the TVM `LICENSE`

Reply:
right, but this will be published in crates.io as a separate independent crate. As this this is what I've seen from other projects.

Reply:
sure, but now it's inside a repo with its own license. You can tell crates what the license is [in the `Cargo.toml`](https://doc.rust-lang.org/cargo/reference/manifest.html#package-metadata)